registration/views.py

```diff
@@ -403,9 +419,11 @@ def render_done(self, form, **kwargs):
         self.storage.reset()
         # also clean out the session
         clean_session(self.request.session)
-        clean_cobrand_session(self.request.session)
-        clean_widget_session(self.request.session)
-        clean_email_source_session(self.request.session)
+        # I Don't think we need to clear those session values on registration done, as it is going to be cleared on the
+        # next visit for Start page anyways.
+        # clean_cobrand_session(self.request.session)
+        # clean_widget_session(self.request.session)
```

Comment:
I'm not suggesting that there would be a problem, I'm just asking to make sure we've both thought through all the implications of not clearing this information at this point.